docs/effects/testing.md

```diff
@@ -104,3 +104,43 @@ describe('My Effects', () => {
   });
 });
 ```
+
+### Effects as functions
+
+Effects can be defined as functions as well as variables. Defining an effect as a function allows you to define default values while having the option to override these variables during tests. This without breaking the functionality in the application.
+
+For example, if we would have an effect that handles a typeahead we may want to debounce the user's input:
```

Comment:
Update to describe the example. 


"The following example effect debounces the user input into from a search action."

Reply:
Thanks for being so patient with me 😅

Reply:
No problem at all 👍